### ISSUE TITLE
Slider announcement does not reflect selected range or updated values

### ISSUE THREAD

**Issue by andyragu** [highcharts/highcharts#25287]
When the sliders are adjusted, an announcement is made similar to the behavior of the Zoom controls. However, the same announcement is read regardless of the slider positions, so it does not accurately reflect the user’s selected range.

In addition, there is no immediate feedback indicating what each slider has been set to. The only way to determine the updated value is to navigate away from the slider and then back to it, at which point the percentage value has changed. Even then, the percentage alone is not especially helpful context.

#### Expected behaviour
The announcement should accurately reflect the selected range after the slider is adjusted.
If the range announcement is accurate and complete, there is no need to separately announce the position of the individual slider.
Users should receive clear and immediate feedback when adjusting the slider.

#### Actual behaviour
Adjusting a slider triggers an announcement.
The announcement does not change based on the slider’s position.
Users do not receive immediate feedback about the selected value or range.
Updated values are only apparent after moving focus away and back to the slider.

**Comment by magdalena-gut** [comment]
Hi @andyragu, thank you for reporting this issue. Could you please provide more specific information on how to reproduce this behavior? A demo example would be greatly appreciated!

**Comment by andyragu** [comment]
Hi @magdalena-gut, here's a link to the best example I could find [https://jsfiddle.net/api/post/library/pure/](url). When using keyboard controls to move the slider to the left or right there is no announcement made to the user to indicate that the slider has been updated

**Comment by magdalena-gut** [comment]
Hi @andyragu , could you please update the link once again - the provided one doesn't work for me. Thanks!

**Comment by andyragu** [comment]
Sorry about that here's a working link: [example](https://www.highcharts.com/demo/stock/responsive). Thank you!
